linux/system/cpu_frequency.sls

```diff
@@ -0,0 +1,10 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.cpu_frequency is defined %}
+
+ondemand_service_disable:
```

Comment:
This is not any generic approach, what's this service about? Eg. such service is not installed on xenial.
Find some more generic approach of setting cpufreq govenor (you probably want to force performance govenor instead of ondemand, right?)